addons/website/models/ir_ui_view.py

```diff
@@ -431,13 +430,10 @@ def _prepare_qcontext(self):
                 ]
 
             qcontext.update(dict(
-                self._context.copy(),
```

Comment:
why do we need the context move?

Reply:
Since the method is now declared in portal and overridden here the context copy is done in portal and website specific infos are added here after the super call. See addons/portal/models/ir_ui_view.py above. If I remember correctly we need the lang param that is in context so the move makes sense to me.

Reply:
> Since the method is now declared in portal and overridden here the context copy is done in portal and website specific infos are added here after the super call. See addons/portal/models/ir_ui_view.py above.

I think we understood that @fja-odoo :') 

> If I remember correctly we need the lang param that is in context so the move makes sense to me.

That's indeed probably it. We could maybe only copy the lang parameter and leave the rest out... do we need to investigate more @JKE-be ? Seems not a big deal to me, could even be done in base IMO... but I don't see all the impacts.